src/meta.ts

```diff
@@ -39,9 +41,16 @@ export class Meta {
     this.tags = tcl.Transform(inputs.tags);
     this.flavor = fcl.Transform(inputs.flavor);
     this.date = new Date();
+    this.commit_date = this.getCommitDate();
     this.version = this.getVersion();
   }
 
+  private getCommitDate(): Date {
+    const myOutput = execSync('git show -s --format="%ci" HEAD');
```

Comment:
Also we don't want to exec `git` when the `context` is `workflow`. You should just grab the `commits.timestamp` from the GitHub API for this case.